### ISSUE TITLE
Using base root instead (to CDN cases)

### ISSUE THREAD

**Issue by tassioFront** [jarlef/vite-plugin-preload#6]
Hi,

I have noticed that vite-plugin-preload does not respect the `base` URL config.

I have tested changing[ this line](https://github.com/jarlef/vite-plugin-preload/blob/main/src/index.ts#L42) and it works as expected. It happens because the plugin uses `base` from the `server` field (`viteConfig.server.base`), but the `base` URL comes from the root instead (`viteConfig.base`).

Current behavior

```HTML
 <link
      rel="modulepreload"
      href="/assets/chunk-name.e5987366.js" 
    />
```

Expected behavior

```HTML
 <link
      rel="modulepreload"
      href="https://my-cudn.domain.com/assets/chunk-name.e5987366.js" 
    />
```

Does make sense to open a pull request?

**Comment by tassioFront** [comment]
@jarlef, are you maintaining this repo yet? I'm asking because see old issues and pull requests.

**Comment by jarlef** [comment]
Sorry about the late reply. I will take a look at it.

**Comment by jarlef** [comment]
Fix has been published in latest version (0.3.0)

**Comment by tassioFront** [comment]
Thanks @jarlef
